src/azure-cli/azure/cli/command_modules/appconfig/_validators.py

```diff
@@ -76,7 +76,7 @@ def validate_export(namespace):
         if (namespace.dest_name is None) and (namespace.dest_connection_string is None):
             raise CLIError("usage error: --config-name NAME | --connection-string STR")
     elif destination == 'appservice':
-        if namespace.appservice_name is None:
+        if namespace.appservice_account is None:
```

Comment:
What is the difference between appservice_name and appservice_account?

Reply:
This was a bug. The argument we want to check for is --appservice-account. We don't have anything like appservice-name.